ansible/roles/rhacm_assisted_installer/files/assisted-installer.yml

```diff
@@ -0,0 +1,37 @@
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  name: assisted-installer
+  namespace: assisted-installer
```

Comment:
Does the `Namespace` already exist?

Reply:
In this environment, (a baremetal cluster deployed by assisted-installer) the asssited-installer namespace exists already.